html/modules/custom/reliefweb_import/src/Plugin/ReliefWebImporter/InoreaderImporter.php

```diff
@@ -549,7 +550,10 @@ protected function processDocumentData(string $uuid, array $document): array {
     $url = $document['canonical'][0]['href'];
 
     // Retrieve the title and clean it.
-    $title = $this->sanitizeText(html_entity_decode($document['title'] ?? ''));
+    $title = $this->sanitizeText(html_entity_decode($document['title'] ?? ''), TRUE);
```

Comment:
Why the `TRUE`? This will preserve new lines but that's not something we allow for titles usually.

Reply:
Fixed